src/features/DocumentFormatter.ts

```diff
@@ -110,14 +111,17 @@ class PSDocumentFormattingEditProvider implements DocumentFormattingEditProvider
         range: Range,
         options: FormattingOptions,
         token: CancellationToken): TextEdit[] | Thenable<TextEdit[]> {
-        return this.executeRulesInOrder(document, range, options, 0);
+
+        let textEdits: Thenable<TextEdit[]> = this.executeRulesInOrder(document, range, options, 0);
+        AnimatedStatusBar.setAnimatedStatusBarMessage("formatting", textEdits);
```

Comment:
I'd recommend a more obvious string like "PowerShell code formatting in progress"